contract-addresses/mainnetAddresses.ts

```diff
@@ -1311,6 +1311,11 @@ const MainnetAddresses: MainnetAddresses = {
     address: '0x27Fae9E49AD955A24bB578B66Cdc962b5029fbA9',
     category: AddressCategory.Deprecated
   },
+  stwBulkHarvest: {
```

Comment:
Please keep these ordered by category + alphabetical. This belongs in the Rewards section